zerver/tests/test_decorators.py

```diff
@@ -284,6 +284,7 @@ def my_webhook_raises_exception(request: HttpRequest, user_profile: UserProfile)
 
 {body}
                 """
+            message = message.strip(' ')
```

Comment:
What's the story with this change?

Reply:
@timabbott: The message template can have trailing whitespace due to the indentation before the ending `"""`, we just strip trailing whitespace so that tests don't fail because of differences in just that. :)